beacon-chain/core/helpers/validator_churn.go

```diff
@@ -20,23 +20,30 @@ import (
 //	    )
 //	    return churn - churn % EFFECTIVE_BALANCE_INCREMENT
 func BalanceChurnLimit(activeBalance primitives.Gwei) primitives.Gwei {
+	cfg := params.BeaconConfig()
 	churn := max(
-		params.BeaconConfig().MinPerEpochChurnLimitElectra,
-		(uint64(activeBalance) / params.BeaconConfig().ChurnLimitQuotient),
+		cfg.MinPerEpochChurnLimitAlpaca,
```

Comment:
change `MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA` to `MIN_PER_EPOCH_CHURN_LIMIT_ALPACA`  would be better for consistency